src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Self\Plugin;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Pantheon\Terminus\Exceptions\TerminusNotFoundException;
+
+/**
+ * Installs a Terminus plugin using Composer.
+ * @package Pantheon\Terminus\Commands\Self\Plugin
+ */
+class InstallCommand extends PluginBaseCommand
+{
+    const ALREADY_INSTALLED_MESSAGE = '{project} is already installed.';
+    const COMPOSER_INSTALL_COMMAND =
+        'composer create-project --stability=%s --prefer-source --keep-vcs -n -d %s %s:~%s';
+    const INVALID_PROJECT_MESSAGE = '{project} is not a valid Packagist project.';
+    const USAGE_MESSAGE = 'terminus self:plugin:<install|add> <Packagist project 1> [Packagist project 2] ...';
+
+
+    /**
+     * Install one or more Terminus plugins.
+     *
+     * @command self:plugin:install
+     * @aliases self:plugin:add
+     *
+     * @param array $projects A list of one or more plugin projects to install
+     * @option string $stability Version stability such as stable, beta, alpha, etc.
+     *
+     * @usage <Packagist project 1> [Packagist project 2] ...
+     */
+    public function install(array $projects, $options = ['stability' => 'stable',])
+    {
+        foreach ($projects as $project) {
```

Comment:
Shouldn't you throw an error immediately if there are no projects provided with a usage message?  This will let the user know there should be at least one project argument.

Reply:
If no projects are provided, the validation hook in the subsequent function will catch it.

Reply:
Gotcha.  I see the code for that now in lines 53-55.